src/main/java/Main/Main.java

```diff
@@ -42,5 +42,7 @@ public static void main(String[] args) throws InterruptedException {
         }
 
         System.out.println("TEST");
+        System.out.println("TEST2");
+        System.out.println("TEST 3 ");
```

Comment:
This is violating SOLID.

Reply:
Single responsibility
Open–closed
Liskov substitution
Interface segregation
Dependency inversion